docs/usage/modules/manager/.pages

```diff
@@ -0,0 +1,8 @@
+title: Managers
```

Comment:
### Weird order in sidebar for manager intro page

I'm checking out the preview on the Renovate docs PR and the sidebar is confusing:

![manager-intro-out-of-order](https://user-images.githubusercontent.com/34918129/205616942-ea74dda3-bc04-4521-ba12-3ef9ca8808ed.png)

The page title is "Introduction", and it's in the wrong place in the sidebar.

### Order in which Renovate executes

I think Renovate follows this ordering in the execution of the program:

1. Platforms
2. Managers
3. Datasources
4. Versioning

### Correct sidebar idea

We should probably match the order given above in the docs.

Maybe the title for the `managers.md` needs to be changed in the `docs/usage/modules/manager/index.md` file?

Reply:
👉 
![image](https://user-images.githubusercontent.com/1798109/205620719-88fc911e-1c42-4688-968a-7484878f7637.png)
natural sort order